bin/version-manager.js

```diff
@@ -90,10 +90,24 @@ function run(files, patterns) {
   const mode = cmd.major ? 'major' : cmd.patch ? 'patch' : 'minor'
 
   // Create our test structures.
-  const viewer =
-    process.env.TRAVIS || cmd.print === 'simple'
-      ? new printers.SimplePrinter(files, { refresh: 100 })
-      : new printers.PrettyPrinter(files, { refresh: 100 })
+  let viewer
+  switch (cmd.print) {
```

Comment:
I have omitted the `TRAVIS` check because I don't think we utilize that service any longer. I can work it back in if I'm wrong.